rfc021-platform-support-policy.md

```diff
@@ -65,12 +62,15 @@ Tier 2 supported platforms are those on which Omnitruck will serve packages, but
 * AIX 5.1L
 * FreeBSD 8
 * NetBSD
+* OpenBSD (i386 and amd64)
 * Windows 2003R2, Windows 2003, Windows 2000
 * RHEL/CentOS/Oracle/Scientific 4.x or older
 * RHEL or SLES on POWER (ppc64) or System/z
 * HP-UX
 * Mac OS X < 10.9, anything ppc
 * Debian < 7.0, anything ppc or arm
+* Gentoo Linux (rolling release)
+* Arch Linux (rolling release)
```

Comment:
These aren't mentioned in install.sh, not sure why they were in T2 to start with. :+1: